src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -171,8 +196,6 @@
 
 /* Between-page links and captions */
 nav.bd-links {
-  margin-right: -1rem;
-
```

Comment:
I couldn't figure out why this rule was here but it was clipping the focus ring, so I removed it. I asked @12rambau if he could remember why he added it and if I could remove it, and he replied, ["I guess the only way to find out is to remove it 😄"](https://github.com/pydata/pydata-sphinx-theme/pull/726#discussion_r1377936652)